db/migrate/20240109113642_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: { to_table: :posts }
```

Comment:
- [ ] [Optional] There is no need to add `foreign_key: { to_table: :users }` and `:post, foreign_key: { to_table: :posts }` because Active Record will add the foreign keys as `user_id` and `post_id` automatically.
```suggestion
      t.references :user
      t.references :post
```